openassessment/workflow/api.py

```diff
@@ -261,9 +261,11 @@ def update_from_assessments(submission_uuid, assessment_requirements):
 
     """
     workflow = _get_workflow_model(submission_uuid)
+    staff_required = assessment_requirements.get("staff", {}).get('staff', {}).get('required', False)
+    is_staff_override = workflow.staff_score_exists() and not staff_required
```

Comment:
I think it might make sense to have force_override be explicit on the api level as well, and then call the api from `staff_assess` in the staff assessment mixin.